python/paddle/tensor/logic.py

```diff
@@ -372,3 +375,35 @@ def not_equal(x, y, name=None):
     """
     out = fluid.layers.not_equal(x, y, name=name, cond=None)
     return out
+
+
+def is_tensor(x):
+    """
+
+    This function tests whether input object is a paddle.Tensor or a paddle.ComplexTensor.
```

Comment:
Forget to say in last comment, since this API doc will also represent on official website, could you build a preview screenshot and attach on this PR?

Not it is LGTM for code, but I'm afraid of Doc.

Reply:
Discussed offline, now the 2.0 API may have to change FluidDoc to show correct English Doc page. So we won't put English Doc preview here but in the next FluidDoc PR.